olm/src/jsTest/kotlin/io/github/matrixkt/olm/runTest.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.matrixkt.olm
+
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.promise
+
+
+actual fun BaseTest.runTest(block: () -> Unit): dynamic = GlobalScope.promise {
+    JsOlm.init().then { block() }
+}
```

Comment:
I don't think the tests are running still. Don't promises need to be awaited?

I think this should be written like this.
```suggestion
actual fun BaseTest.runTest(block: () -> Unit): dynamic = JsOlm.init().then { block() }
```

Reply:
let me check. Tests were running and failing when something was bad but maybe this was some kind of side effect.